src/api/.cdsrc.json

```diff
@@ -1 +1,17 @@
-{}
+{
+    "cds": {
+        "requires": {
+            "GENERATIVE_AI_HUB": {
+                "kind": "rest",
+                "credentials": {
+                    "destination": "SMART_CONVERTER_AI_CORE",
+                    "requestTimeout": "300000"
+                },
+                "RESOURCE_GROUP_ID": "default",
+                "DEPLOYMENTS": {
+                    "CHAT_COMPLETION_ID": "d6bb1fe78f97a16f"
```

Comment:
- please make this generic and put as "src/api/.cdsrc.sample.json"
- rename destination to GENERATIVE_AI_HUB"
- .cdsrc.json should be in gitignore